schema/Member.re

```diff
@@ -0,0 +1,45 @@
+open Graphql_lwt;
+open GraphqlHelpers;
+open Library;
+
+type t = {
+  id: string,
+  email: string,
+};
+
+let resolver =
```

Comment:
this is actually the type right

Reply:
```suggestion
let typ =
```

Reply:
is not the type resolver?

Reply:
the resolver which I mean in this case is the root guy that fetches the initial data and "rains down" to its field types